modules/codegen/src/main/scala/com/twilio/guardrail/generators/Http4sServerGenerator.scala

```diff
@@ -60,7 +60,7 @@ object Http4sServerGenerator {
             List(combinedRouteTerms),
             List.empty,
             methodSigs,
-            renderedRoutes.flatMap(_.supportDefinitions),
+            renderedRoutes.flatMap(_.supportDefinitions).groupBy(_.structure).map(_._2.head).toList, // Only unique supportDefinitions by structure
```

Comment:
This could still cause name collisions, as the bodies could be different. Thoughts around prefixing support definitions by their associated operationId?

Reply:
@blast-hardcheese That would be preferable indeed,

it's easy to prefix the generated `*Var` with the `operationId` in `com.twilio.guardrail.generators.Http4sServerGenerator.ServerTermInterp#generatePathParamExtractors`, however

I can't find a neat way to prefix the endpoint matcher (`case req @ GET ...`) with `operationId`, AFAIK the path matchers are generated in `com.twilio.guardrail.SwaggerUtil.paths#generateUrlHttp4sPathExtractors`, but I don't have access to the RouteMeta in there and it doesn't feel right to pass the route meta object all the way from `com.twilio.guardrail.generators.Http4sServerGenerator.ServerTermInterp#generateRoute` into the Swagger utils. Any ideas on how to handle this?

Reply:
Iirc when I was reviewing yesterday, it may be a decent strategy to pass a common prefix into the support generator method, bubbling up what other parameters use to a single definition, then propagating out.